static/styles/settings.css

```diff
@@ -97,19 +97,6 @@ h3 .fa-question-circle-o {
     }
 }
 
-.user-avatar-section {
-    float: right;
-
-    .avatar-controls {
-        margin-top: 20px;
-        box-shadow: none;
-    }
-
-    .inline-block {
-        box-shadow: 0 0 10px hsla(0, 0%, 0%, 0.1);
-    }
-}
-
```

Comment:
Removed this from `settings.css` and added to `Image_upload_widget.css` since it belongs to `Image_upload_widget.css`.